src/oc/hocdec.h

```diff
@@ -120,11 +117,11 @@ typedef struct Symbol {	/* symbol table entry */
 	char	*name;
 	short	type;
 	short	subtype;	/* Flag for user integers */
-#if defined(__cplusplus)
+//#if defined(__cplusplus)
```

Comment:
purge

Reply:
Actually I want to keep that there for the time being, so that people don't say WTF. I will add a comment.